examples/push_results_to_girder.py

```diff
@@ -0,0 +1,52 @@
+## The purpose of this script is to be added to the end of the testing
+## done as part of ITKPerformanceBenchmarking
+## Hopefully we can set it up so that dashboard that run the
```

Comment:
`so that dashboard that run`: `so that dashboards that run`
or else
`so that the dashboard that runs`

Reply:
Done.